pages/loggedout/index.js

```diff
@@ -1,5 +1,8 @@
 export default function LoggedOut() {
   return (
-    <div>You are not currently logged in. Please login via Single View.</div>
+    <>
+      <title>Logged Out</title>
```

Comment:
`title` is [not valid as a body element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title), it needs to be inside of `head`. There is a Next.js component that will put things into the page head for you, called (surprisingly enough) [Head](https://nextjs.org/docs/api-reference/next/head).

Reply:
nice!